platform/build-faiss/build-prereqs.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+export PATH=$HOME/.local/bin:$PATH
+export DEBIAN_FRONTEND=noninteractive
+
+sudo -E apt-get update && sudo -E apt-get dist-upgrade -y
+
+# Install python and build essentials and essential libraries
+sudo -E apt-get install -y python3-venv python3-pip python3-dev build-essential libssl-dev libffi-dev lib
```

Comment:
Lines 10 and 11 should be concatenated.

Additionally, `-E` is unnecessary. I don't know why `apt-get` would ever need to inherit the child's environment.